packages/widget/src/index.ts

```diff
@@ -1,5 +1,6 @@
 import './styles/global.css';
 export { SwapWidgetProvider } from './provider';
 export { SwapWidget, SwapWidgetProps } from './ui';
+import './ui/WebComponent';
```

Comment:
just added it here temporarily but I think it's better to have the web-component loader in a separate import